src/components/Dropdown/dropdown.module.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&family=Roboto:wght@100;300;400;500;700;900&display=swap');
```

Comment:
why here in this specific file

Reply:
Will look for different way to import it

Reply:
i have removed import cause we can use that font without importing